server/budgettracker.web/Controllers/BudgetController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using budgettracker.business.Services;
+using budgettracker.business.Api.Contracts.BudgetApi;
+using System;
+
+namespace budgettracker.web.Controllers
+{
+    [Route("api/budget")]
+    public class BudgetController: Controller
+    {
+        private readonly IBudgetService _budgetService;
+
+        public BudgetController(IBudgetService budgetService )
+        {
+            _budgetService = budgetService;
+        }
+
+        [HttpPost]
+        [Route("create")]
+        public async Task<ActionResult<BudgetResponseContract>> CreateBudget(BudgetResquestContract budget)
+        {
+            await _budgetService.CreateBudget(budget);
+
+            return Ok();
```

Comment:
What if it fails?

Reply:
For consistancy sake, the Api should always return an ApiResponse. not just a 200 response. The ApiResponse should return the results in all cases so, in this case, it should return the resulting Budget model AFTER it was created in the Database (in case any changes were made to it on creation such as Id generation or cleaning up of data).

Reply:
Yeah that is another part I am looking to up now!